docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+services:
+  mongo:
+    build:
+      context: .
+      dockerfile: ./docker/dockerfile.mongodb-replica.yml
+    container_name: plasmidi-mongo
+    ports:
+      - '27018:27018'
+    networks:
+      - plasimidi-network
+    volumes:
+      # seed data can be created at ./init-mongo.js
+      # - ./init-mongo.js:/docker-entrypoint-initdb.d/init-mongo.js
+      # volume to persist data locally
+      - mongo-data:/data/db
+    restart: always
+    logging:
+      options:
+        max-size: 1g
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: mongoadmin
+      MONGO_INITDB_ROOT_PASSWORD: secret
+      MONGO_REPLICA_PORT: 27018
+  server:
+    container_name: plasmidi-server
+    depends_on:
+      - mongo
+    build:
+      context: .
+      dockerfile: ${DOCKER_ENVFILE_PATH:-docker/dockerfile.dev.yml}
+    environment:
+      MONGODB_URL: mongodb://mongoadmin:secret@mongo:27018/plasmidi?directConnection=true&authSource=admin
+      MONGO_INITDB_ROOT_USERNAME: mongoadmin
+      MONGO_INITDB_ROOT_PASSWORD: secret
+    networks:
+      - plasimidi-network
+    ports:
+      - "3000:3000"
+      - "3001:3001"
+    volumes:
+      - ./:/src
+networks:
+  plasimidi-network:
+    driver: bridge
+volumes:
+  mongo-data:
```

Comment:
this volume persists your local data. not required but makes life easy by not having to remake users all the time and stuff.